### ISSUE TITLE
Help Documentation around Templates

### ISSUE THREAD

**Issue by brentonmallen1** [NotePlan/plugins#86]
**Is your feature request related to a problem? Please describe.**
Templates are very useful and powerful but not very approachable.  To understand how to use them, one has to dig into github and search a few different locations to get full context on how to implement some things.

**Describe the solution you'd like**
I'd like to see some approachable documentation in the `Advanced` section of the help site [here](https://help.noteplan.co/category/38-tips-tricks).  Something with a few examples that people who might not be programmers or comfortable sifting through github to figure out how to use them.

**Describe alternatives you've considered**
Currently, the alternative is to simply go through documentation in github for the particular plugin one would like to implement  in a template.  As a programmer, this isn't daunting but I can see it being a blocker for others to implement which is a shame as I think templates can be very powerful.

**Comment by dwertheimer** [comment]
Yep. This is true. Templates may be a special case where it deserves a spot in the NP documentation. My only concern is that it’s constantly updating so hard to keep the NP docs current.

**Comment by jgclark** [comment]
For Templates themselves, @brentonmallen1, there's just the one place I linked you to. It's the details on the various {{commands}} which are provided by different plugins, and are therefore spread around. There's an active debate between the main plugin developers at the moment about how to more cleanly separate the two.

@EduardMe controls the Help site. As @dwertheimer says the plugins, including the Templates, are still a bit of a moving target. It might also be the case that some aspects get migrated into the main app itself.

But, yes, at the very least we could do with a new Help page on Templates and an advanced section that gives some examples of the {{commands}} and points to them.
